homeassistant/components/minecraft_server/quality_scale.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration doesn't provide any service actions.
+  appropriate-polling:
+    status: done
+    comment: Default polling interval is set to 60s.
+  brands: done
+  common-modules: done
+  config-flow: done
```

Comment:
We should not have the name configurable. Rather we should set the name of the entry to something the user can recognize and rename themselves

Reply:
Oh lol, you're asking the name but then set the entry title to the address

Reply:
Alright, thanks. I'll have to check that. Setting to **todo** for now.